web/onos-gui/src/app/onos-topo/devices-list/devices-list.component.css

```diff
@@ -18,11 +18,24 @@
  ONOS GUI -- Topology Device View -- CSS file
  */
 
-#ov-devices .tabular-header {
+ #ov-devices .tabular-header {
     text-align: left;
 }
 
-#ov-devices div.summary-list .table-header td {
+
+
+#ov-devices div.summary-list .table-sticky thead, tbody tr {
+    display: table;
+    width: 95%;
```

Comment:
this is nice - i hadn't considered putting the width on the `tr` - it works well - I think you can make it 100%

Reply:
done